contents/docs/product-analytics/identify.mdx

```diff
@@ -81,133 +81,106 @@ The ID returned is either the ID automatically generated by PostHog or the ID th
+```flutter
+// The alias will be associated with the current distinct ID
+posthog().alias("alias_id")
 ```
 
 </MultiLanguage>
 
-There are two requirements when assigning an `alias_id`:
-
-1. It cannot be associated with more than one `distinct_id`.
-2. The `alias_id` **must not** have been previously used as the `distinct_id` argument of an `identify()` or `alias()` call. For example: Assume we previously called `posthog.identify('distinct_id_one')`. It is not possible to use `distinct_id_one` as an alias ID:
-
 <MultiLanguage selector="tabs">
 
-```node
-// Assume we previously identified a user with 'user1' using posthog.identify('user1')
-// You should not use user1 as an alias for user2
-client.alias({
-    distinctId: 'user2',
-    alias: 'user1',
-})
-```
+<h4>Server-side</h4>
 
-```js-web
-// Assume we previously identified a user with 'user1' using posthog.identify('user1')
-// You should not use user1 as an alias for user2
+```node
 client.alias({
-    distinctId: 'user2',
-    alias: 'user1',
+    distinctId: 'distinct_id',
+    alias: 'alias_id',
 })
 ```
 
 ```python
-# Assume we previously identified a user with 'user1' using posthog.identify('user1')
-# You should not use user1 as an alias for user2
-posthog.alias('user2', 'user1');
+posthog.alias(previous_id='distinct_id', distinct_id='alias_id')
 ```
 
 ```php
-# Assume we previously identified a user with 'user1' using posthog.identify('user1')
-# ❌ The following is not possible:
-# You cannot use user1 as an alias for user2
 PostHog::alias(array(
-  'distinctId' => 'user2',
-  'alias' => 'user1'
+  'distinctId' => 'distinct_id',
+  'alias' => 'alias_id'
 ));
 ```
 
 ```ruby
-# Assume we previously identified a user with 'user1' using posthog.identify('user1')
-# You should not use user1 as an alias for user2
-posthog.alias({
-  distinct_id: "user2",
-  alias: "user1",
-})
+posthog.alias(
+  distinct_id: "distinct_id",
+  alias: "alias_id"
+)
 ```
 
 ```go
-// Assume we previously identified a user with 'user1' using posthog.identify('user1')
-// You should not use user1 as an alias for user2
 client.Enqueue(posthog.Alias{
-  DistinctId: "user2",
-  Alias: "user1",
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
 })
 ```
 
 ```java
-// Assume we previously identified a user with 'user1' using posthog.identify('user1')
-// You should not use user1 as an alias for user2
-posthog.alias("user2", "user1");
+posthog.alias("distinct_id", "alias_id");
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "distinct_id": "distinct_id",
+        "alias": "alias_id"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "event": "$create_alias"
+}' <ph_client_api_host>/i/v0/e/
 ```
 
 </MultiLanguage>
 
+The two IDs are merged with the following rules:
+
+- If neither the `distinct_id` nor the `alias_id` are already identified, a new person is created with both IDs.
```

Comment:
One new person is created with which ID? Or two new people are created with both of the IDs?

Reply:
<img width="1199" alt="Screenshot 2025-06-27 at 12 33 04 PM" src="https://github.com/user-attachments/assets/7faabe38-819b-4a2b-8a2b-162667bbee58" />
<img width="1199" alt="Screenshot 2025-06-27 at 12 34 13 PM" src="https://github.com/user-attachments/assets/962d1b93-1bc1-422e-975f-fe4d35b412cc" />

It's not deterministic as far as I can tell 🙃 

From this code:
```js
for (let i = 0; i < 10; i++) {
    createUser(`user-${i}-on-web`)
    createUser(`user-${i}-on-mobile`)
    createUser(`user-${i}-on-tablet`)
    createAlias(`user-${i}-on-web`, `user-${i}-on-tablet`)
    createAlias(`user-${i}-on-mobile`, `user-${i}-on-tablet`)
}
```